src/libstd/process.rs

```diff
@@ -253,6 +253,10 @@ impl Command {
     /// Builder methods are provided to change these defaults and
     /// otherwise configure the process.
     ///
+    /// The `PATH` from the Command's environment will be searched
+    /// if the program contains only a single path component, as for
+    /// `execvp`.
```

Comment:
Yeah I was wondering whether that would help make the behavior clearer to people or if it was better style not to mention it.  I can remove that phrase.